hbase-endpoint/src/test/java/org/apache/hadoop/hbase/client/TestRpcControllerFactory.java

```diff
@@ -85,32 +86,15 @@ public HBaseRpcController newController(final List<CellScannable> cellIterables)
 
     private static Multiset<Integer> GROUPED_PRIORITY = ConcurrentHashMultiset.create();
     private static AtomicInteger INT_PRIORITY = new AtomicInteger();
-    private static AtomicInteger TABLE_PRIORITY = new AtomicInteger();
```

Comment:
For what purpose, we need to remove the table priroirty counter ?   don't understand here ...

Reply:
We do not call the setPriority(TableName) method any more in async client, we will decide the priority outside the RpcController and then set with the setPriority(int) method. This is why we need to change the UT itself.